sos-centar-api/SosCentar.BusinessLogic/Services/UserService.cs

```diff
@@ -19,5 +22,81 @@ public bool ValidateUserCredentials(string email, string password)
 		{
 			return _userRepository.ValidateUserExists(email, _securityService.HashPassword(password));
 		}
+
+		public User GetUserIfValid(string email, string password)
+		{
+			return _userRepository.GetUserIfValid(email, _securityService.HashPassword(password));
+		}
+
+		public IEnumerable<ResponseUserDto> GetAllUsers()
+		{
+			var users = _userRepository.GetAll();
+			var userDtos = new List<ResponseUserDto>();
+
+			foreach (var user in users)
+			{
+				var userDto = new ResponseUserDto()
+				{
+					Email = user.Email,
+					FirstName = user.FirstName,
+					LastName = user.LastName,
+					IsAdmin = user.IsAdmin
+				};
+				userDtos.Add(userDto);
+			}
+
+			return userDtos;
+		}
+
+
+		public ResponseUserDto GetUserByEmail(string email)
+		{
+			var user = _userRepository.GetByEmail(email);
+
+			if (user == null)
+			{
+				throw new KeyNotFoundException("There is no user with email: " + email);
+			}
```

Comment:
this can be simplified. e.g.
```c#
var user = _userRepository.GetByEmail(email) ?? throw new KeyNotFoundException($"There is no user with email: {email}");
```

And its better to check for null with `is` (`is not`) keyword rather than `==` (`!=`), because `==` (`!=`) can be overriden and `is` can't